frontend/components/dashboard/table/DashboardTableSummary.vue

```diff
@@ -167,12 +123,13 @@ const searchPlaceholder = computed(() =>
         </h1>
         <BcDropdown
           v-if="tableIsShown"
-          v-model="selectedTimeFrame"
+          :model-value="timeFrame"
           :options="timeFrames"
           option-value="id"
           option-label="name"
           class="small"
           :placeholder="$t('dashboard.group.selection.placeholder')"
+          @select="($event) => { emit('update', $event, props.query) }"
```

Comment:
you did tightly couple us here to an implementation detail of `prime vue` here....🤔

Reply:
this might already work with `@update`